module/config.mjs

```diff
@@ -2054,6 +2054,32 @@ DND5E.advancementTypes = {
   Trait: advancement.TraitAdvancement
 };
 
+/* -------------------------------------------- */
+/*  Sources                                     */
+/* -------------------------------------------- */
+
+/**
+ * List of books available as sources.
+ * @enum {string}
+ */
+DND5E.sourceBooks = {
+  "SRD 5.1": "SOURCE.BOOK.SRD"
+};
+preLocalize("sourceBooks", { sort: true });
+
+/* -------------------------------------------- */
+
+/**
+ * List of licenses available with sources.
+ * @enum {string}
+ */
+DND5E.sourceLicenses = {
```

Comment:
I've thought about it some more and it just doesn't feel like a list of licenses is something that really belongs in the system implementation. Are we going to add to this? Are modules expected to add their own licenses to this list? Since CC-BY-4.0 is the only one included by default, is that going to be taken as some sort of implicit recommendation that people use this license for their content?

I don't think it's a huge deal but, unless you have strong objections, I might suggest we remove this and replace it with a plain text field for now. We can very easily add it back in later if we need to since it would not require a migration.